Content.Server/Singularity/EntitySystems/ContainmentFieldSystem.cs

```diff
@@ -34,8 +35,8 @@ private void HandleFieldCollide(EntityUid uid, ContainmentFieldComponent compone
 
         if (TryComp<PhysicsComponent>(otherBody, out var physics) && physics.Mass <= component.MaxMass && physics.Hard)
         {
-            var fieldDir = Transform(uid).WorldPosition;
-            var playerDir = Transform(otherBody).WorldPosition;
+            var fieldDir = _transformSystem.GetWorldPosition(Transform(uid));
+            var playerDir = _transformSystem.GetWorldPosition(Transform(otherBody));
```

Comment:
```suggestion
            var fieldDir = _transformSystem.GetWorldPosition(uid);
            var playerDir = _transformSystem.GetWorldPosition(otherBody);
```

Reply:
oh ok, I can get rid of all the Transform()s